### DISCUSSION TITLE
Subcutaneous dosing for large  molecule

### DISCUSSION THREAD

**Discussion by zjh112** [Open-Systems-Pharmacology/Forum#1372] [Questions & Problems]
Hi,
I am working on subcutaneous PK data of a large molecule and am trying to do PBPK modeling. My question is how to do subcutaneous dosing in PK-Sim and Mobi, is the same to the small molecule? I would really appreciate the help.

Thanks.

**Comment by prvmalik** [comment]
How complicated is it? Is pre-systemic elimination mechanistically relevant to your research question? Is it an ADC?

**Reply by zjh112** [reply]
Yes, it is ADC. I want to study monoclonal antibody, but some monoclonal antibodies were administered subcutaneously.

**Reply by prvmalik** [reply]
Ok, so you will have to model local and pre-systemic degradation followed by release of the payload and diffusion of the payload into plasma. This requires optimization of 2-3 parameters. I can upload a schematic structure that I have used in the past in a bit.

**Reply by zjh112** [reply]
Thank you very much. So I need to add a new compartment?

**Reply by YangYu-China** [reply]
> Ok, so you will have to model local and pre-systemic degradation followed by release of the payload and diffusion of the payload into plasma. This requires optimization of 2-3 parameters. I can upload a schematic structure that I have used in the past in a bit.

Could you show me a screenshot? Thank you very much
